cfmtoolbox/plugins/random_sampling.py

```diff
@@ -11,12 +11,12 @@
 
 
 @app.command()
-def random_sampling(model: CFM, amount: int = 1) -> CFM:
+def random_sampling(model: CFM, number: int = 1) -> CFM:
```

Comment:
this sounds weird?, is it really easy to understand from this name that it is the parameter for how many samples you get?

Reply:
What about something like `--num-samples`?

Reply:
oki :)